lib/exercises.js

```diff
@@ -1,33 +1,57 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(1)
```

Comment:
This is O(n) time complexity, where n is the number of words. Yes, querying and updating `anagramDict` is O(1), but we have to do it n times.